app/src/main/java/com/example/keniphhomemaintenance/MainActivity.kt

```diff
@@ -1,38 +1,32 @@
 package com.example.keniphhomemaintenance
 
+import android.content.res.Configuration
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
+import androidx.compose.material.*
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.tooling.preview.Preview
+import com.example.keniphhomemaintenance.maintenance_screen.MaintenanceCard
+import com.example.keniphhomemaintenance.maintenance_screen.MaintenanceItem
+import com.example.keniphhomemaintenance.navigation.Navigation
 import com.example.keniphhomemaintenance.ui.theme.KeniphHomeMaintenanceTheme
 
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContent {
             KeniphHomeMaintenanceTheme {
-                // A surface container using the 'background' color from the theme
                 Surface(color = MaterialTheme.colors.background) {
-                    Greeting("Android")
                 }
+                Configuration.UI_MODE_NIGHT_YES
```

Comment:
does this check for night mode or automatically turn on night mode?

Reply:
I was messing with night mode when the scaffold turned everything blue. So I don't believe I need this anymore, you right.